### PR TITLE
create olathe file with a few recommendations

### LINKED ISSUE #14: Things to do in Olathe, KS
# Olathe, KS

It would be nice if there were a list of recommendations of things to do in Olathe, KS. 

👍 💃 🎉 
🍔

### PR DESCRIPTION
**Pull request by githubteacher**
### Description of Change

Adding a file for Olathe, KS 
This fixes #14 

### Test results

No testing was done

### Who should know about the change?

Anyone going to Kansas :) 

/cc: @githubteacher

### CODE REVIEW

File: briana-olathe-ks.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Olathe, KS
+
+### Things to do
+- Royals Baseball
+- Kauffman Center for the Performing Arts
```
Comment (flogat) [inline briana-olathe-ks.md:5]: You cannot forget Joey's Pizza!

Comment (githubteacher) [inline briana-olathe-ks.md:5]: What about nature?

**Review by githubteacher — COMMENTED** [review]
Looks good 👍

**Review by flogat — CHANGES_REQUESTED** [review]
Please include Joey's  as discussed

File: briana-olathe-ks.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Olathe, KS
+
+### Things to do
+- Royals Baseball
```
Comment (if338) [inline briana-olathe-ks.md:4]: What about a football match once

**Comment by githubteacher** [comment]
@florian-shellfire I made some changes, what do you think?

**Review by flogat — APPROVED** [review]
Enjoy the Pizza!

**Pull request merged**